hashes/zkevm/src/keccak/README.md

```diff
@@ -0,0 +1,75 @@
+# ZKEVM Keccka
+## Vanilla
+Keccak circuit in vanilla halo2. This implementation starts from Scroll's version then adopts some changes from [this PR](https://github.com/scroll-tech/zkevm-circuits/pull/216) and [PSE version](https://github.com/privacy-scaling-explorations/zkevm-circuits/tree/main/zkevm-circuits/src/keccak_circuit).
```

Comment:
this did _not_ start from Scroll's version. Scroll's version is also based on the PSE version, which is due to Brecht. I had the correct attribution here: https://github.com/axiom-crypto/halo2-lib/blob/70c5cc05f8e7e0904533ae4044191dcd16bea214/hashes/zkevm-keccak/src/lib.rs#L2